### PR TITLE
COMMERCE-10147 - OM

### PR DESCRIPTION
**Pull request by brianikim**
https://issues.liferay.com/browse/COMMERCE-10147

@luca-pellizzon

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:stop

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java [hunk 1]
```diff
@@ -493,12 +494,22 @@ public List<CommerceOrder> getUserPendingCommerceOrders(
 			long companyId, long groupId, String keywords, int start, int end)
 		throws PortalException {
 
+		return getUserPendingCommerceOrders(
+			companyId, groupId, keywords, start, end, null);
+	}
+
+	@Override
+	public List<CommerceOrder> getUserPendingCommerceOrders(
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:502]: I don't like pending here. Pending is a specific status, so you can maybe call it getUserOpenCommerceOrders

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:502]: Done.

**Comment by brianikim** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:commerce

**Comment by brianikim** [comment]
[LocalFile.SFCommerceOrders#AssertDateAndTimeAreDisplayedAsOrderDate](https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/287775//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SFCommerceOrders_AssertDateAndTimeAreDisplayedAsOrderDate_) - [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-12/test-1-31/test-portal-acceptance-pullrequest(master)/6360/functional-tomcat90-mysql57-jdk8/1/1/LocalFile.SFCommerceOrders_AssertDateAndTimeAreDisplayedAsOrderDate/index.html.gz) - [Poshi Summary](https://testray.liferay.com/reports/production/logs/2022-12/test-1-31/test-portal-acceptance-pullrequest(master)/6360/functional-tomcat90-mysql57-jdk8/1/1/LocalFile.SFCommerceOrders_AssertDateAndTimeAreDisplayedAsOrderDate/summary.html.gz) - [Console Output](https://testray.liferay.com/reports/production/logs/2022-12/test-1-31/test-portal-acceptance-pullrequest(master)/6360/functional-tomcat90-mysql57-jdk8/1/1/jenkins-console.txt.gz)

@baharturk ,

Is the testcase correct in producing its expected value? I see the poshi summary:
`
Actual text "Dec 14, 2022, 12:47:02 AM" does not contain expected text (case insensitive) "Dec 14, 2022 0" at "//div[contains(@class,'table-list-title')]/a[contains(.,'')]/following::div[4]"
`

Actual task:
`
		task ("Then the order date and time is shown") {
			var currentDate = DateUtil.getFormattedCurrentDate("MMM d, yyyy K", "UTC");

			AssertTextEquals.assertPartialText(
				key_orderId = "",
				locator1 = "CommerceOrders#PLACED_ORDERS_LIST_DATE",
				value1 = "${currentDate}");
		}
`

I am not sure how to reconstruct that K value to match the pattern we are expecting.

cc/ @victoroscuro

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:commerce

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/3263

**Pull request closed without merging**